assets/css/_sass/website/_layout.scss

```diff
@@ -23,6 +23,9 @@ nav.side {
     left: 0;
     bottom: 0;
     overflow-y: auto;
+    a {
+      font-weight: 400;
+    }
```

Comment:
This is scoped to only target only `<a>` tags inside a  `nav` element with `.side` class applied.